### ISSUE TITLE
[QST]Does the version 22.10 support the orc+zstd/parquet+zstd

### ISSUE THREAD

**Issue by lw-cxn** [NVIDIA/spark-rapids#6567]
Does the version 22.10 support the orc+zstd/parquet+zstd ? This format is widely used and urgently needs to be supported.

**Comment by tgravescs** [comment]
yes we support zstd for reading for orc and parquet since the 22.08 release.
zstd is supported in Apache spark 3.2.0+ versions
